analyzer/consensus/consensus.go

```diff
@@ -347,8 +349,24 @@ func (m *processor) ProcessBlock(ctx context.Context, uheight uint64) error {
 }
 
 func (m *processor) queueBlockInserts(batch *storage.QueryBatch, data *consensusBlockData) error {
+	var cmtMeta cometbft.BlockMeta
+	if err := cbor.Unmarshal(data.BlockHeader.Meta, &cmtMeta); err != nil {
+		m.logger.Warn("could not unmarshal block meta, may be incompatible version",
+			"height", data.BlockHeader.Height,
+			"err", err,
+		)
+		// We only try to unmarshal into the current version of the metadata
+		// structure (oasis-core Eden + CometBFT at time of writing). This may
+		// fail on blocks from an incompatible earlier version. Skip indexing
+		// the block metadata in that case.
+	}
+
+	var proposerAddr *string
+	if cmtMeta.Header != nil {
+		proposerAddr = common.Ptr(cmtMeta.Header.ProposerAddress.String())
```

Comment:
Can we convert this to an oasis-address (assuming we just feed the bytes to a bech32 derivation) and store it like that instead? It'd be nice to avoid another address format and to keep things consistent, but it'd also be v helpful for supporting queries on signed blocks/proposed blocks/uptime for a given validator.

Same for the `prevSigners` addresses below

Also, does the `proposerAddr` address refer to the node_id (type `base64_ed25519_pubkey` in the db), but encoded in hex?